docs/scalafix-migrations.md

```diff
@@ -13,10 +13,13 @@ Here are two pull requests that demonstrate this feature:
 And here are all pull requests where Scala Steward applied Scalafix
 migrations:
 
+<!-- These links exists but returns 404 often so skip check -->
```

Comment:
The `pulls` URLs require that the user is logged into GitHub. I guess this is why they are reported as dead.

Reply:
Aha, I see